File: programmers/lev1/82612.py
def solution(price, money, count):
    answer = 0

    sum_num = 0

    for i in range(count):
        sum_num += (price * (i + 1))

    if sum_num >= money:
        answer = sum_num - money
    else:
        answer = 0

    return answer


price = 3
money = 20
count = 4
solution(price, money, count)

File: java/src/programmers/year2023/lv1/_82612.java
package programmers.year2023.lv1;

public class _82612 {
    public static void main(String[] args) {
        int price = 3;
        int money = 20;
        int count = 4;

        System.out.println(solution(price, money, count));
    }

    public static long solution(int price, int money, int count) {
        long totalPrice = 0;
        for (int i = 1; i <= count; i++) {
            totalPrice += (price * i);
        }

        return totalPrice - money > 0 ? totalPrice - money : 0;
    }
}
